cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgreSQLSink.java

```diff
@@ -283,8 +290,8 @@ public void expirateRecords(long expirationTime) throws CygnusExpiratingError {
      */
     private abstract class PostgreSQLAggregator {
 
-        // string containing the data fieldValues
-        protected String aggregation;
+        // object containing the aggregted data
```

Comment:
aggregted -> aggregated

Reply:
Fixed in [3eb9f3c](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/3eb9f3c58274e03d78f55acc4b8ffa54e4b30e7c)